subsys/caf/modules/sensor_data_aggregator.c

```diff
@@ -23,6 +23,23 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_CAF_SENSOR_MANAGER_LOG_LEVEL);
 #define __DEFINE_DATA(i, agg_id, size)							\
 	static uint8_t agg_ ## agg_id ## _buff_ ## i ## _data[size] __aligned(4);
 
+#if (CONFIG_EVENT_MANAGER_PROXY)
```

Comment:
no - please do not do it, we cannot add dependency on event manager proxy here
we can check if there is a property with memory region in dts I guess